pkg/instrumentation/build.go

```diff
@@ -0,0 +1,9 @@
+package instrumentation
```

Comment:
@srikanthccv can we find a better place for these settings?

Reply:
Let's move the build info to `pkg/version/build`?

Reply:
Moved this to `pkg/version`